client/blocks/eligibility-warnings/support-link.tsx

```diff
@@ -0,0 +1,77 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { HelpCenter } from '@automattic/data-stores';
+import { Button } from '@wordpress/components';
+import {
+	useDispatch as useDataStoreDispatch,
+	useSelect as useDateStoreSelect,
+} from '@wordpress/data';
+import { localize, LocalizeProps } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import ActionPanelLink from 'calypso/components/action-panel/link';
+import { getSectionName } from 'calypso/state/ui/selectors';
+import type { HelpCenterSelect } from '@automattic/data-stores';
+
+const HELP_CENTER_STORE = HelpCenter.register();
+
+const SupportLink = ( {
```

Comment:
Handles switching between the standard contact link and the help assistant button, plus activating the help assistant.